frontend/apiversions.go

```diff
@@ -5,6 +5,7 @@ package main
 
 // This will invoke the init() function in each
 // API version package so it can register itself.
+// HERE
```

Comment:
Tests are failing because the apiVersions aren't getting populated because this part isn't being imported when testing. 

What's the right way to deal with this? AFAIK, in Classic we explicitly pass the valid versions when creating the NewFrontend

Reply:
Can't you copy the below import into your test?

Reply:
I could just import that in the test file or frontend.go but that seems wrong. We'll need to (potentially, so even worse) update those files every time we want to add a version.

Hmmm